react/features/mobile/polyfills/browser.js

```diff
@@ -108,6 +108,14 @@ function _visitNode(node, callback) {
         global.addEventListener = () => {};
     }
 
+    // Promise.allSettled is supported from RN 0.63 onwards, use a polyfill for that.
+    // Invokes its shim method to shim Promise.allSettled if it is unavailable or noncompliant.
+    //
+    // Required by:
+    // lib-jitsi-meet/JitsiConference.js
+    global.allSettled = require('promise-allSettled');
```

Comment:
Is it necessary to assign it to global? Doesn't `require('promise-allSettled').shim();` do it?